### PR TITLE
Addresses #1736 NTR 'valve interstitial cell'

### PR DESCRIPTION
**Pull request by bvarner-ebi**
#1736

### CODE REVIEW

**Comment by bvarner-ebi** [comment]
#gogoeditdiff

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -2797,6 +2798,7 @@ Declaration(Class(obo:CP_0000039))
 Declaration(Class(obo:CP_0000040))
 Declaration(Class(obo:CP_0000043))
 Declaration(Class(obo:D96882F1-8709-49AB-BCA9-772A67EA6C33))
+Declaration(Class(obo:GO_0070278))
```
Comment (shawntanzk) [inline src/ontology/cl-edit.owl:2801]: I think not the greatest idea to declare a GO term, does protege do this?

**Comment by shawntanzk** [comment]
hmmm strange that that didnt trigger actions... will just review without it now but probably should look into whats going on

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -28983,6 +28984,19 @@ AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4030031 "2022-11-30T1
 AnnotationAssertion(rdfs:label obo:CL_4030031 "interstitial cell")
 EquivalentClasses(obo:CL_4030031 ObjectIntersectionOf(obo:CL_0000003 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005169)))
 
+# Class: obo:CL_4030032 (valve interstitial cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref <https://orcid.org/0000-0001-7655-4833>) Annotation(oboInOwl:hasDbXref "PMID:25311230"^^xsd:string) Annotation(oboInOwl:hasDbXref "https://doi.org/10.1098/rsif.2017.0580"^^xsd:string) Annotation(oboInOwl:hasDbXref "https://doi.org/10.1161/ATVBAHA.120.314789"^^xsd:string) Annotation(oboInOwl:hasDbXref "https://doi.org/10.1161/JAHA.117.006339"^^xsd:string) Annotation(oboInOwl:hasDbXref "https://doi.org/10.2353/ajpath.2007.070251"^^xsd:string) obo:IAO_0000115 obo:CL_4030032 "A cardiac valve cell that resides in a valve leaflet. Along with valve endothelial cells, a valve interstitial cell maintains tissue homeostasis for the function of cardiac valves through secreting biochemical signals, matrix proteins and matrix remodeling enzymes."^^xsd:string)
```
Comment (shawntanzk) [inline src/ontology/cl-edit.owl:28989]: I would prefer the definiendum to be "An interstitial cell
that is part of the cardial valve" given the logical definition.

**Comment by shawntanzk** [comment]
Just to double check @gouttegd  are the changes in the sssom file correct? 

@bvarner-ebi, not a big deal if you rather not change your protocol, and I don't think its bad to make those changes during the pipeline, but if you want to avoid pulling unrelated others to review this, you can undo the changes made to files you don't want to touch :) I'm sure there's some ODK way to not touch those, but I usually just let it do its thing and remove stuff i don't want lol.

**Comment by gouttegd** [comment]
@shawntanzk Yes, the changed in the SSSOM file reflect changes that were made in the past two months on the FBbt side. We released a FBbt version last week, so the changes got picked up here when imports were refreshed.

**Pull request merged**